docs/manual/fread_examples.rst

```diff
@@ -22,73 +44,66 @@ Read text data directly::
      a     b     2
      a     b     3
 
-Read from text file (any file that has the `.read()` method)::
-
-    result = fread('iris.csv')
-    result.head(5)
-
-        sepal_length	sepal_width	petal_length	petal_width	species
-    0	5.1	            3.5	            1.4	            0.2	        setosa
-    1	4.9	            3	            1.4	            0.2	        setosa
-    2	4.7	            3.2	            1.3	            0.2	        setosa
-    3	4.6	            3.1	            1.5	            0.2	        setosa
-    4	5	            3.6	            1.4	            0.2	        setosa
-
 
-Read from url::
+- Read from a url::
 
     url = "https://raw.githubusercontent.com/Rdatatable/data.table/master/vignettes/flights14.csv"
     fread(url)
 
-Read from an archive. If there are multiple files, only the first will be read; you can specify the path to the specific file you are interested in::
+- Read from an archive:
 
+If there are multiple files, only the first will be read; you can specify the path to the specific file you are interested in::
 
     fread("data.zip/mtcars.csv")
 
-Read from `.xlsx` files - `.xls` files are not supported::
+**Note:** Use :func:`iread()` if you wish to read in multiple files in an archive; an iterator of Frames is returned.
+
+- Read from ``.xls`` or ``.xlsx`` files ::
 
     fread("excel.xlsx")
 
 For excel files, you can specify the sheet to be read::
 
     fread("excel.xlsx/Sheet1")
 
-Note: `xlrd <https://pypi.org/project/xlrd/>`_ must be installed to read in excel files.
+**Note:**
+        - `xlrd <https://pypi.org/project/xlrd/>`_ must be installed to read in excel files.
+
+        -  Use :func:`iread()` if you wish to read in multiple sheets; an iterator of Frames is returned.
 
-You can also read in data from the command line. Simply pass the command line statement to the `cmd` parameter::
+- Read in data from the command line. Simply pass the command line statement to the ``cmd`` parameter::
 
      #https://blog.jpalardy.com/posts/awk-tutorial-part-2/
      #You specify the `cmd` parameter
      #Here we filter data for the year 2015
      fread(cmd = """cat netflix.tsv | awk 'NR==1; /^2015-/'""")
 
-The command line can be very handy with large data; you can do some of the preprocessing before reading in the data to `datatable`.
+The command line can be very handy with large data; you can do some of the preprocessing before reading in the data to ``datatable``.
 
 Dealing with Unicode Data
 -------------------------
 
-`Fread` is pretty powerful and can automatically detect the encoding in a number of cases.::
+You can specify the encoding via the ``encoding`` parameter; ``fread`` will recode into ``UTF-8`` before reading::
 
     data = ("num,name\n"
             "1,alïce\n"
             "2,bøb\n"
             "3,cárol\n"
             )
 
-    fread(data)
+    fread(data, encoding = "latin-1")
```

Comment:
The concept of "encoding" means how certain abstract characters are stored at byte level. As such, it makes sense to talk about the encoding of a file (because a file is just a sequence of bytes), or the encoding of a `bytes` object, but there is no such thing as "encoding" of a python string (a python string has an underlying encoding, of course, but it's complicated, and mutable, and invisible to the user).

In short, when input is a string the concept of encoding does not apply. Or at least shouldn't. In your example it will actually garble the input, which is a bug.

Reply:
ok. had a different understanding of encoding. Thanks for clearing that.